osgi.specs/docbook/708/util.features.xml

```diff
@@ -240,13 +241,27 @@ Features may have dependencies on other Features. Features inherit the capabilit
               <entry>Optional</entry>
               <entry>The short descriptive name of the Feature.</entry>
             </row>
+            
+            <row>
+              <entry>categories</entry>
+              <entry>Array of String</entry>
+              <entry>Optional, defaults to an empty array</entry>
+              <entry>The categories this Feature belongs to. The values are user-defined.</entry>
+            </row>
 
             <row>
               <entry>description</entry>
               <entry>String</entry>
               <entry>Optional</entry>
               <entry>A longer description of the Feature.</entry>
             </row>
+            
+            <row>
+              <entry>docURL</entry>
+              <entry>URI</entry>
```

Comment:
Is `URI` a JSON type? I would assume the type here is `String` and, as commented above, let the user decide whether to parse into a URI.